pom.xml

```diff
@@ -339,6 +340,17 @@
             <version>1.1.19</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.elasticsearch.client</groupId>
+            <artifactId>elasticsearch-rest-high-level-client</artifactId>
+            <version>${elasticstack.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>net.sf.jopt-simple</groupId>
+                    <artifactId>jopt-simple</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
Do you have in mind that "Apache HTTP Components" is packaged as a Jenkins Plugin? From what I remember of some Elasticsearch client jars, there is a dependency on Apache HTTP.

Reply:
is a TODO to review if we can free weight from the Elasticsearch client